src/admin/dumb_components/custom_email_form.js

```diff
@@ -1,19 +1,22 @@
 const React = require('react')
-const { map, objOf, zipWith, merge, compose } = require('ramda')
+const { map, objOf, zipWith, merge, compose, indexBy, prop, converge, dissoc } = require('ramda')
 
 export default ({ members, submit }) => {
   const onSubmit = (e) => {
     e.preventDefault()
-    const template = format_message(e.target[0].value)
-    const emailBodies = map(compose(objOf('email_body'), template), members)
-    const emails = zipWith(merge, members, emailBodies)
-    submit(emails)
+    const template = format_message(e.target)
+    const emailBodies = map(compose(objOf('content'), template), members)
+    const emailsArr = zipWith(merge, members, emailBodies)
+    const setEmailKey = map(compose(indexBy, prop), ['primary_email', 'secondary_email'])
```

Comment:
This a) duplicates some functionality from the `emails` reducer and b) includes some functionality that seems like it should really live in the reducer -- or at least the action creator.

I'm sorry I missed this in the previous PR, but it's best to, as much as possible, keep the logic in the `modules` file